lib/wor/requests/base.rb

```diff
@@ -74,7 +74,19 @@ def after_error(response, options)
       end
 
       def uri(path)
-        URI.join(base_url, path)
+        URI.join(formatted_base_url, formatted_path(path))
+      end
+
+      def formatted_base_url
+        return "#{base_url}/" if base_url.last != '/'
```

Comment:
A bit of a bother but you could just do `base_url.last != '/' ? "#{base_url} : base_url`